source/manage-apps/deploy/deployment-models-and-regions.txt

```diff
@@ -70,7 +70,15 @@ Cloud Deployment Regions
 - Sydney (``ap-southeast-2``)
 - Virginia (``us-east-1``)
 
-The following regions are available for :ref:`local deployment
+{+service+} supports deployment and writes in the following Azure regions:
+
+- ``EUROPE_WEST``
```

Comment:
[nit] We may want to use the Azure region name here to match the other lists on this page. 
```
westeurope
eastus2
southeastasia
eastasia
westus
```

Reply:
Updated to match Tim's list!